tests/test_it.py

```diff
@@ -194,6 +194,28 @@ def test_I201_import_mock_config():
     ]
 
 
+def test_I201_most_specific_imports():
+    errors = run_flake8(
+        """
+        import urllib2
+        import urllib2.URLError
```

Comment:
This technically leads to an error, since `URLError` is not a module, but I think it's worth including for completeness. This is definitely a situation that can come up in practice, if an entire submodule of a library needs to be banned. I can update the names to `foo.bar` if that's helpful.